### DISCUSSION TITLE
Add alias for JSON resource

### DISCUSSION THREAD

**Discussion by gjm** [dedoc/scramble#478] [Q&A] [answered]
Having more than one JSON resource with the same name, but with different namespaces, causes "weird" schema names, built using the namespace, to eliminate duplicates.

Is it possible to add an alias or explicitly naming a schema in order to overcome this?

Thanks in advance.

**Comment by romalytvynenko** [comment]
@gjm this is definitely something I will do in the upcoming release. Thank you!

**Comment by Bruksa** [comment]
Any update here?

**Reply by romalytvynenko** [reply]
No updates here.

**Comment by romalytvynenko** [comment, marked as answer]
Implemented in `0.12.x`

https://scramble.dedoc.co/usage/response#explicitly-naming-schemas
